ci/jobs/scripts/git_helper.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
```

Comment:
is it the same file as in ./tests/ci? can we just import it from there?

Reply:
not already the 100% same, it uses `Utils.Shell`, for example.